spec/features/create_listing_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+feature 'create a listing' do
+  scenario 'user enters a name, description and price for their space' do
+    visit('/listings/create')
+    fill_in "name_field", with: "Cute ass cottage"
+    fill_in "desc_field", with: "This cottage is deadass so cute"
+    fill_in "price_field", with: "69.69"
```

Comment:
This shouldn't be a float (and will need some kind of data manipultion before entering into table"